mne/time_frequency/spectrum.py

```diff
@@ -486,6 +498,10 @@ def method(self):
     def nave(self):
         return self._nave
 
+    @property
+    def mt_weights(self):
+        return self._mt_weights
+
```

Comment:
I guess it's even more unlikely a new method would be added with taper weights and weights for something else, so maybe just `weights` instead of `mt_weights` or `taper_weights` is slightly more future-proof??

Reply:
SGTM, `weights` it is.